.github/workflows/tests.yml

```diff
@@ -11,25 +11,23 @@ on:
 
 jobs:
   tests:
-    name: tests
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
```suggestion
    strategy:
      fail-fast: false
```

Reply:
This way, if one of the tests fails (for example node 12), the other tests still run.